nemo/collections/nlp/modules/common/text_generation_server.py

```diff
@@ -156,6 +157,14 @@ def put(self):
             if neighbors < 0:
                 return "num of neighbors must be an integer no less than 0"
 
+        end_strings = ['<|endoftext|>']
```

Comment:
I think the default end_strings should be the empty list, to have nothing. You are already handling <|endoftext|> in your text generation utils by checking for tokenizer.eos_id no?

Reply:
if empty list, it will not stop generating. Now we use text generation strategy to decide when to stop. `['<|endoftext|>']` is the default behavior that uses eos_id to stop.